astropy/__init__.py

```diff
@@ -255,11 +255,11 @@ def online_help(query):
         )
     ):
         # The last clause in the the above disjunction deserves explanation:
-        # When using relative imports like ``from .. import config``, the
+        # When using relative imports like ``from astropy import config``, the
```

Comment:
This change should not happen! (Or the whole comment needs adjusting)

Reply:
rolling it back.